public/playersBetController.js

```diff
@@ -0,0 +1,140 @@
+"use strict";
+export default class playersBetController {
+  constructor(board) {
+    this.currentPlayerTurn = 0;
+    this.board = board;
+    this.currentBet = 0;
+
+    this.controlsContainerEle = document.querySelector("#game");
+  }
+
+  //Each player must place their bet before the cards are dealt
+
+  //After dealing the cards if the dealer's card is Ace, the players can make an insurance bet, the maximum that can be bet is half the initial bet, it is left on the insurance bet line and it pays 2 to one
+  // if an insurance bet is made, then the player starts his hand normally, that is, with the same options as always
+
+  controlsHtml() {
+    return `
+    <div class='bet-controls'>
+        <div class='bet-controls__pot'>Pot: ${this.activePlayer.pot}</div>
+        <div class='bet-controls__bet'>Current bet: ${this.currentBet}</div>    
+        <button type='button' class= 'bet-controls__btn btn--25'>
+        25
+        </button>
+        <button type='button' class= 'bet-controls__btn btn--50'>
+        50
+        </button>
+        <button type='button' class= 'bet-controls__btn btn--100'>
+        100
+        </button>    
+        <button type='button' disabled class= 'bet-controls__btn btn--clear'>
+        Clear
+        </button>
+        <button type='button' disabled class= 'bet-controls__btn btn--bet'>
+        Confirm bet
+        </button>
+    </div>    
+      `;
+  }
+
+  rederBetControls() {
```

Comment:
There is a typo here:
```suggestion
  renderBetControls() {
```